packages/react-core/src/components/Form/examples/FormFieldGroups.tsx

```diff
@@ -0,0 +1,324 @@
+          <TextInput isRequired id="3-label2" name="3-label2" value={inputValues['3-label2']} onChange={handleChange} />
+        </FormGroup>
+        <FormFieldGroup
+          header={
+            <FormFieldGroupHeader
+              titleText={{
+                text: 'Nested field group 1 (non-expandable)',
+                id: 'nested-field-group1-non-expandable-titleText-id'
+              }}
+            />
+          }
+        >
+          <FormGroup label="Label 1" isRequired fieldId="group4-label1">
+            <TextInput
+              isRequired
+              id="group4-label1"
+              name="group4-label1"
+              value={inputValues['group4-label1']}
+              onChange={handleChange}
+            />
+          </FormGroup>
+          <FormGroup label="Label 2" isRequired fieldId="group4-label2">
+            <TextInput
+              isRequired
+              id="group4-label2"
+              name="group4-label2"
+              value={inputValues['group4-label2']}
+              onChange={handleChange}
+            />
+          </FormGroup>
+        </FormFieldGroup>
+        <FormFieldGroup
+          header={
+            <FormFieldGroupHeader
+              titleText={{
+                text: 'Nested field group 2 (non-expandable)',
+                id: 'nested-field-group2-non-expandable-titleText-id'
+              }}
+              titleDescription="Field group 2 description text."
+            />
+          }
+        >
+          <FormGroup label="Label 1" isRequired fieldId="group5-label1">
+            <TextInput
+              isRequired
+              id="group5-label1"
+              name="group5-label1"
+              value={inputValues['group5-label1']}
+              onChange={handleChange}
+            />
+          </FormGroup>
+          <FormGroup label="Label 2" isRequired fieldId="group5-label2">
+            <TextInput
+              isRequired
+              id="group5-label2"
+              name="group5-label2"
+              value={inputValues['group5-label2']}
+              onChange={handleChange}
+            />
+          </FormGroup>
+        </FormFieldGroup>
+      </FormFieldGroupExpandable>
+      <FormFieldGroup
+        header={
+          <FormFieldGroupHeader
+            titleText={{ text: 'Field group 4 (non-expandable)', id: 'field-group4-non-expandable-titleText-id' }}
+            titleDescription="Field group 4 description text."
+            actions={
+              <>
+                <Button variant="link">Delete all</Button> <Button variant="secondary">Add parameter</Button>
+              </>
+            }
+          />
+        }
+      >
+        <FormGroup label="Label 1" isRequired fieldId="exapnded-group2-label1">
+          <TextInput
+            isRequired
+            id="exapnded-group2-label1"
+            name="exapnded-group2-label1"
+            value={inputValues['exapnded-group2-label1']}
```

Comment:
We should make sure the non-expandable example doesn't have a field with `expanded` in the name!

Reply:
sounds good! I'll get right on that :D